index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header-box">
+        <p>John Doe</p>
+        <img src="/images/Menu-icon.png" alt="" />
+      </header>
```

Comment:
- [ ] Nice job with wrapping the header/toolbar with the `header` tag. You're doing well 💯